frontends/mit-open/src/components/FieldAvatar/FieldAvatar.tsx

```diff
@@ -73,6 +77,10 @@ const AvatarImg = styled.img<AvatarStyleProps>`
     imageVariant === "inverted" ? "filter: saturate(0%) invert(100%);" : ""}
   height: ${({ imageSize }) => IMG_SIZES[imageSize]};
   width: auto;
+  ${({ theme }) => theme.breakpoints.down("sm")} {
+    height: auto;
+    width: 100%;
```

Comment:
I don't think `width: 100%` is what we want here. It's OK for some of the wider logos, but not great for narrow ones:

<img width="722" alt="Screenshot 2024-06-25 at 1 50 23 PM" src="https://github.com/mitodl/mit-open/assets/9010790/90aee678-e901-464b-9f95-127f7282538a">

`max-width: 100%` seems better, but just changing width to max-width made the image disappear for me. I do not entirely understand why.

This worked OK:

```diff
index 9749fa40f..a8330c2a2 100644
--- a/frontends/mit-open/src/components/FieldAvatar/FieldAvatar.tsx
+++ b/frontends/mit-open/src/components/FieldAvatar/FieldAvatar.tsx
@@ -61,10 +61,6 @@ const AvatarContainer = styled.div<AvatarStyleProps>`
   align-items: center;
   height: ${({ imageSize }) => IMG_SIZES[imageSize]};
   width: auto;
-  ${({ theme }) => theme.breakpoints.down("sm")} {
-    height: auto;
-    width: 100%;
-  }
 `
 const AvatarImg = styled.img<AvatarStyleProps>`
   display: flex;
@@ -78,8 +74,8 @@ const AvatarImg = styled.img<AvatarStyleProps>`
   height: ${({ imageSize }) => IMG_SIZES[imageSize]};
   width: auto;
   ${({ theme }) => theme.breakpoints.down("sm")} {
-    height: auto;
-    width: 100%;
+    flex: 1;
+    max-width: 100%;
   }
 `
```
though I do not know i it's the best solution.

**Suggestion:** Additionally, I would remove `display: flex` and the flex-related properties (direction, align, justify) from `AvatarImg`. The element doesn't have children, so they aren't doing anything.

Reply:
I think i got the logo sizing to work using some css calc magic. also removed flex styles